README.md

```diff
@@ -5,14 +5,16 @@
 Read the API reference [here](https://pub.dev/documentation/genius_lyrics/latest/).
 
```

Comment:
It would be interesting to add a table of contents here as the number of sections just grew. But this is not a requirement for the merge to happen, I can always add it later.